pyproject.toml

```
@@ -57,5 +57,5 @@ per-file-ignores = """
 """
 
 [tool.codespell]
-skip = './docs/_build,./docs/_static,./taxonomy.yaml'
+skip = './docs/_build,./docs/_static,./taxonomy.yaml,./repo/*.patch'
```

```suggestion
skip = './docs/_build,./docs/_static,./taxonomy.yaml,./repo/**/*.patch'
```

Does the previous single `*.patch` work locally for you for patches in packages? I'd think you'd have to do a double star to get it to recurse into the packages, but I could be wrong / they might be using a different glob library.